packages/create-package/create-package.js

```diff
@@ -39,7 +32,7 @@ const isNPMRC = sourcePath => {
 // Validation of user input
 const ComponentNamePattern = /^[A-Z][A-Za-z0-9]+$/;
 const NamespacePattern = /^[a-z][a-z0-9\.\-]+$/;
-const isNameValid = name => typeof name === "string" && name.match(/^[a-zA-Z][a-zA-Z0-9\-_]+$/);
+const isNameValid = name => typeof name === "string" && name.match(/^(@[a-z0-9-~][a-z0-9-._~]*\/)?[a-z0-9-~][a-z0-9-._~]*$/);
```

Comment:
let's rename it to isPackageNameValid  as only name does not exactly explains which name